vars/karmaExecuteTests.groovy

```diff
@@ -9,20 +9,20 @@ import groovy.transform.Field
 
 @Field String STEP_NAME = 'karmaExecuteTests'
 @Field Set GENERAL_CONFIG_KEYS = [
-    'containerPortMappings', //port mappings required for containers. This will only take effect inside a Kubernetes pod, format [[containerPort: 1111, hostPort: 1111]]
-    'dockerEnvVars', //envVars to be set in the execution container if required
-    'dockerImage', //Docker image for code execution
-    'dockerName', //name of the Docker container. If not on Kubernetes pod, this will define the network-alias to the NPM container and is thus required for accessing the server, example http://karma:9876 (default).
-    'dockerWorkspace', //user home directory for Docker execution. This will only take effect inside a Kubernetes pod.
+    'containerPortMappings', /** port mappings required for containers. This will only take effect inside a Kubernetes pod, format [[containerPort: 1111, hostPort: 1111]] */
```

Comment:
The javadoc comments needs to be before the documented entity - like it is for javadoc in general. Having the docu in the same line after the documented entity will not work with our toolset - and would also not work with javadoc (in case of normal java classes).